src/pages/iou/request/step/IOURequestStepAmount.tsx

```diff
@@ -102,14 +103,14 @@ function IOURequestStepAmount({
 
     useFocusEffect(
         useCallback(() => {
-            focusTimeoutRef.current = setTimeout(() => textInput.current?.focus(), CONST.ANIMATED_TRANSITION);
+            focusTimeoutRef.current = setTimeout(() => moneyRequestAmountInput?.current?.focus(), CONST.ANIMATED_TRANSITION);
             return () => {
                 if (!focusTimeoutRef.current) {
                     return;
                 }
                 clearTimeout(focusTimeoutRef.current);
             };
-        }, []),
+        }, [moneyRequestAmountInput]),
```

Comment:
Why is this needed as a dependency? Refs do not change

Reply:
When I remove this, the `react-complier` check fails.